src/promptflow/promptflow/_sdk/entities/_connection.py

```diff
@@ -22,7 +22,7 @@
     ConnectionType,
     CustomStrongTypeConnectionConfigs,
 )
-from promptflow._sdk._errors import UnsecureConnectionError
+from promptflow._sdk._errors import UnsecureConnectionError, SDKError
```

Comment:
Do we have an estimate of how much unknown category percentage will be improved?

Reply:
90% of UNKNOW will be optimized to UserError.